RecoLocalTracker/SiPhase2Clusterizer/python/phase2TrackerClusterizer_cfi.py

```diff
@@ -3,7 +3,7 @@
 # Clusterizer options
 siPhase2Clusters = cms.EDProducer('Phase2TrackerClusterizer',
     src = cms.InputTag("mix", "Tracker"),
-    maxClusterSize = cms.uint32(8),
+    maxClusterSize = cms.uint32(0), # was 8
```

Comment:
if I'm not mistaken, this is a bogus parameter now (unless VERIFY_PH2_TK_CLUS is set).
It may be worth to leave a note about it here.